compiler/test/compilable/test23013.d

```diff
@@ -0,0 +1,43 @@
+// https://issues.dlang.org/show_bug.cgi?id=24013
+
+struct PropDescriptor(T)
+{
+    void define(T delegate() aGetter, string aName) {}
+}
+
+enum Get;
```

Comment:
not required anymore after your reduction

Reply:
This is used to attribute the function. A better reduction could probably be made, I think this one is fine.